android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -114,11 +113,8 @@ class CustomTabsToolbarFeature(
     internal fun init(config: CustomTabConfig) {
         // Don't allow clickable toolbar so a custom tab can't switch to edit mode.
         toolbar.display.onUrlClicked = { false }
-
-        // If it's available, hold on to the readable colour for other assets.
-        if (updateToolbarBackground && config.toolbarColor != null) {
-            readableColor = getReadableTextColor(config.toolbarColor!!)
-        }
+        readableColor =
+            config.toolbarColor?.let { getReadableTextColor(it) } ?: toolbar.display.colors.menu
```

Comment:
there's apparently some unit tests around this value. It looks like your CI is all green, but it might be worthwhile adding a unit test to cover the use path this fix is for.
[CustomTabsToolbarFeatureTest](https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/feature/customtabs/src/test/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeatureTest.kt)

Reply:
Good catch, updated tests 👍